packages/trader/src/App/Components/Elements/VideoPlayer/video-player.tsx

```diff
@@ -148,15 +154,15 @@ const VideoPlayer = ({ src, is_mobile, data_testid }: TVideoPlayerProps) => {
         progress_bar_filled_ref.current.style.setProperty('width', `${new_width}%`);
         video_ref.current.currentTime = (Number(video_ref.current.duration) * new_width) / 100;
 
-        timer_ref.current = setTimeout(() => {
+        play_on_rewind_timeout.current = setTimeout(() => {
             video_ref?.current?.play();
             setIsPlaying(true);
             setIsAnimated(true);
             setIsEnded(false);
-        }, 500);
+        }, 100);
```

Comment:
Reduced the time in order to make animation smoother. Result (observe the way the draggable dot is moving while the user is dragging it, smoothly, not sharply):

https://github.com/kate-deriv/deriv-app/assets/103177211/9f26d232-4653-47e8-b411-b0c1c55a9554